packages/flame/lib/src/game/base_game.dart

```diff
@@ -68,12 +68,15 @@ class BaseGame extends Game with FPSCounter {
   /// This is overwritten to consider the viewport transformation.
   ///
   /// Which means that this is the logical size of the game screen area as
-  /// exposed to the canvas after viewport transformations.
+  /// exposed to the canvas after viewport transformations and camera zooming.
```

Comment:
These dartdocs are not available on `size` if you put them on the `_sizeBuffer`. And as this dartdoc starts with `This is overwritten..` I think we should move this whole block down to `size`...

Reply:
Fixed